docs/lib/auth/auth-events.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Auth Events
+description: Listen to various auth events
+---
+
+AWS Cognito Auth Plugin sends important events through the Amplify Hub.
+
```

Comment:
This needs to be removed, otherwise, JS will show a section called "Auth events"